MC/config/common/pythia8/utils/mkpy8cfg.py

```diff
@@ -52,6 +66,9 @@
 
 args = parser.parse_args()
 
+# derive the process
+args.process = derive_process(args.process, args.idA, args.idB)
```

Comment:
is there also the same for background processes?

Reply:
This script is called for the signal as well for the background. So there is no difference and it would insert the process for signal as well as background. See for instance https://github.com/AliceO2Group/O2DPG/pull/1416/files#diff-d3d7f9e5d4c04202b033167540929ff40471636b8a1bb1ffdbecd1066c051eeaR414